Dockerfile.dev

```diff
@@ -4,7 +4,7 @@ WORKDIR /app
 ENV POSTGRES_VERSION=14
 
 RUN apt update -qq && \
-  apt-get install nodejs build-essential curl git pkg-config libpq-dev libclang-dev postgresql-common -y && \
+  apt-get install nodejs build-essential curl git pkg-config libpq-dev libclang-dev postgresql-common postgresql-client -y && \
```

Comment:
Was this needed? normally `/usr/share/postgresql-common/pgdg/apt.postgresql.org.sh -i -v $POSTGRES_VERSION` should install it

Reply:
it was not working for me 🤔

Reply:
ah wrong file, I will update it!

Reply:
It's still here I think?

Reply:
done!